src/client/Microsoft.Identity.Client/IClientApplicationBase.cs

```diff
@@ -43,7 +43,8 @@ public partial interface IClientApplicationBase
         Task<IEnumerable<IAccount>> GetAccountsAsync();
 
         /// <summary>
-        /// Get the <see cref="IAccount"/> by its identifier among the accounts available in the token cache.
+        /// Get the <see cref="IAccount"/> by its identifier among the accounts available in the token cache and of the same
+        /// instance (authority host) as <see cref="Authority"/> 
```

Comment:
We do not use the term `instance` in MSAL. We use `environment`. See IAccount interface.

```suggestion
        /// environment (authority host) as <see cref="Authority"/> 
```